interface.js

```diff
@@ -0,0 +1,75 @@
+const ul = document.getElementById('articles');
+const fullArticle = document.getElementById('full-article');
+const guardianURL = 'https://content.guardianapis.com/search?api-key=19158819-ba32-4e1f-a54c-d75a65fad4ab&show-fields=starRating,headline,thumbnail,short-url'
+
+function createNode(element) {
+  return document.createElement(element);
+}
+ 
+function append(parent, el) {
+  return parent.appendChild(el);
+}
+
+fetch(guardianURL)
+  .then((resp) => resp.json())
+
+  .then(function(data){
+    var articles = data.response.results;
+    return articles.map(function(article) {
+      let li = createNode('li');
+      
+      let link = createNode('a');
+      link.setAttribute("href", `${article.webUrl}`);
+      link.setAttribute("onClick", `showFullArticle()`);
+      link.setAttribute("class", "viewPage");
+      link.innerHTML = `${article.webUrl}`;
+
+      let headline = createNode('h3');
+      headline.innerHTML = `${article.webTitle}`;
+
+      let summary = createNode('p');
+      var articleApiUrl = "http://news-summary-api.herokuapp.com/aylien?apiRequestUrl=https://api.aylien.com/api/v1/summarize?url=" + `${article.webUrl}`
+
+      let image = createNode('img')
+      image.setAttribute("class", "img-fluid");
+      image.src = `${article.fields.thumbnail}`;
+
+      const fetchPromise = 
+      fetch (articleApiUrl)
+      fetchPromise.then(response => {
+        // console.log(fetchPromise)
+        return response.json();
+      }).then(data => {
```

Comment:
perhaps the response.json could be saved in the variable data in the return statement before it is used in the then call to make it clear what data is referring to